Tests/eZ/Publish/FieldType/TweetTest.php

```diff
@@ -0,0 +1,199 @@
+<?php
```

Comment:
I see. It is a good point. But I wouldn't give too much weight to it, and would disregard it for now.